.kokoro/tests/run_tests.sh

```diff
@@ -168,7 +168,7 @@ test_prog="${PROJECT_ROOT}/.kokoro/tests/run_single_test.sh"
 
 btlr_args=(
     "run"
-    "--max-cmd-duration=60m"
+    "--max-cmd-duration=70m"
```

Comment:
I think updating this is causing every test to run, which is the cause of many errors unrelated to this PR

Reply:
that makes sense

Reply:
Let's try reverting this to 60m, tests shouldn't run for that long anyways and we'll get rid of the "run all the tests" issue.